Motorcycle.js

```diff
@@ -1,6 +1,18 @@
 var Car = require('./Car.js');
 
 //Implement your Motorcycle here
-function Motorcycle() {};
+function Motorcycle(make, model, year, color, seats, passengers) {
+	Car.call(this, make, model, year, color, seats, passengers);
```

Comment:
This looks good, but let's add a way to override seats if the user puts in more than 2:
```
if (this.seats > 2) {
		this.seats = 2;
	}
```